File: PN24.java
// Template for probplems on Numbers

import java.lang.*;
import java.util.*;

class Number
{
    private int iNo;

    public void Accept()
    {
        Scanner sobj = new Scanner(System.in);
        System.out.println("Enter number : ");
        this.iNo = sobj.nextInt();
    }

    public void Display()
    {
        System.out.println("Value is : "+this.iNo);
    }
    
	public int SumEven()
	{
     int iDigit = 0;
	 int iSum = 0;
	
	 if(iNo < 0)
	 {
		iNo = -iNo;
	 }
	 while(iNo > 0)
	 {
		iDigit = iNo % 10;
		if((iDigit % 2) == 0)
		{
			iSum = iSum + iDigit;
		}
		iNo = iNo/10;
	 }
	 return iSum;
	}
}

class PN24
{
    public static void main(String b[])
    {
            Number nobj = new Number();
            int iRet = 0;
			
            nobj.Accept();
            nobj.Display();

            iRet = nobj.SumEven();
			
			System.out.println("The sum of even digits is : "+iRet);
    }
}

File: PN24.cpp


#include<iostream>
using namespace std;

class Number
{
    private:
        int iNo;       

    public:
        int iCnt = 0;
		
        void Accept()       
        {
            cout<<"Enter the value : "<<endl;
            cin>>this->iNo;
        }

        void DisplayF()      
        {
			 for(iCnt = 1; iCnt<= iNo; iCnt++)
             {
                 cout<<iCnt<<endl;        
             }
        }

        int CountFactor()      
        {
			int iFactCnt = 0;
			 for(iCnt = 2; iCnt <= (iNo/2); iCnt++)
             {
                 if(iNo % iCnt == 0)
				 {
					 iFactCnt++;
				 }					 
             }
			 return iFactCnt;
        }
		
		bool CheckPrime()
		{
		   int iRet = 0;
		   iRet = CountFactor();
		   if(iRet = 0)
		   {
			  return true;
		   }
		   else
		   {
			  return false;
		   }
		}

};

int main()
{
    Number nobj;
	
	bool bRet;


    nobj.Accept();
    nobj.DisplayF();

    bRet = nobj.CheckPrime();
	
	if(bRet == true)
	{
		cout<<"it prime number: "<<endl;
	}
	else
	{
		cout<<"it not prime number: "<<endl;
	}


    return 0;
}